test/integration/operator_olm_test.go

```diff
@@ -15,19 +15,21 @@
 package e2e
 
 import (
+	"context"
 	"fmt"
 	"os"
 	"path/filepath"
 	"testing"
 	"time"
 
 	apimanifests "github.com/operator-framework/api/pkg/manifests"
-	operator "github.com/operator-framework/operator-sdk/internal/olm/operator"
-	"github.com/operator-framework/operator-sdk/internal/util/k8sutil"
-
 	operatorsv1alpha1 "github.com/operator-framework/api/pkg/operators/v1alpha1"
 	"github.com/stretchr/testify/assert"
 	apiextv1beta1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
+
+	operator "github.com/operator-framework/operator-sdk/internal/olm/operator"
+	operator2 "github.com/operator-framework/operator-sdk/internal/operator"
```

Comment:
There's some related work going on with `run bundle` in the `github.com/operator-framework/operator-sdk/internal/operator` package. 

The thought is to do the new `run` design in the new package, and once that's mostly there we can port the existing `run packagemanifests` code (mainly the registry setup) into that package.

See https://github.com/operator-framework/operator-sdk/issues/3626